examples/node-selection/README.md

```diff
@@ -0,0 +1,120 @@
+## Node selection example
```

Comment:
We'd also mention that pinging pod to specific machine using this approach is discouraged.  This is a scheduling problem, not something user needs to take care of.   The plan is to add more resource type for node, and more resource requirement for pod.